teps/0027-https-connection-to-triggers-eventlistener.md

```diff
@@ -0,0 +1,135 @@
+---
+title: https-connection-to-triggers-eventlistener
+authors:
+  - "@savitaashture"
+creation-date: 2020-10-19
+last-updated: 2020-10-19
+status: proposed
+---
+
+# TEP-0024: HTTPS Connection To Triggers EventListener
```

Comment:
nit: I think TEP-0024 is already taken. So this should be something else 😛

Reply:
Oh in the file name i gave `TEP-0027` but in the description its TEP-0024 :laughing: :see_no_evil: 

Modified :+1: